src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -74,7 +88,7 @@ object JsonProtocol {
   ))
 
   /** Construct Json formatter for annotations */
-  def jsonFormat(tags: Seq[Class[_ <: Annotation]]) = {
+  def jsonFormat(tags: Seq[Class[_]]) = {
```

Comment:
I don't fully understand why this change is necessary?

Reply:
The classes for which we want to provide tags  for may no longer be a subclass of Annotation, since there may be arbitrarily nested class instances _within_ the annotation we're trying to serialize.